doc/changes/changes_0.1.0.md

```diff
@@ -1,8 +1,15 @@
-# transformers-extension 0.1.0, released XXXX-XX-XX
+# Transformers Extension 0.1.0, released 2022-09-23
 
+Code name: Add a downloader UDF and a set of prediction UDFs using the transformers API
 
-## Summary
 
+## Summary
+This is the initial release of the transformers-extension which provides a 
+downloader UDF that allows us to cache the pre-trained machine learning model 
```

Comment:
Is this really a cache? I see it more as pre-loading existing models. A cache would be in my eyes if the model was auto-generated once and then kept in a cache without user interaction.

Reply:
Yes and no. It would be an cache, except that is not suitable for how storage works in the UDFs. And, on the other hand, we use the cache facilities of the transformers library. Usually, the cache would also include eviction, which is actually not part of the cache functionality of the transformers library. However, users if the libraries probably now about the cache functionality and are used to the term. 

Maybe, we should write something like preload the model to the cache.

Reply:
:heavy_check_mark: